File: englising-be/src/main/java/org/englising/com/englisingbe/singleplay/dto/response/LyricDto.java
package org.englising.com.englisingbe.singleplay.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class LyricDto {
    private Long lyricId;
    private Boolean isBlank;
    private Float startTime;
    private Float endTime;
    private List<String> lyric;
}

File: englising-recommend/dto/lyric_dto.py
from typing import Optional, List
from pydantic import BaseModel


class LyricDto(BaseModel):
    lyricId: Optional[int]
    isBlank: Optional[bool] = False
    startTime: Optional[float] = None
    endTime: Optional[float] = None
    lyric: Optional[List[str]] = None